t/apicast-policy-http-proxy.t

```diff
@@ -1143,3 +1143,139 @@ POST /test?user_key=
 --- no_error_log env
 ["[error]",
 "using proxy: $TEST_NGINX_HTTP_PROXY "]
+
+
+=== TEST 18: MTLS connection to upstream via proxy failed
+--- configuration random_port env
+{
+  "services": [
+    {
+      "backend_version":  1,
+      "proxy": {
+        "api_backend": "https://test-upstream.lvh.me:$TEST_NGINX_RANDOM_PORT",
+        "proxy_rules": [
+          { "pattern": "/", "http_method": "GET", "metric_system_name": "hits", "delta": 2 }
+        ],
+        "policy_chain": [
+          {
+            "name": "apicast.policy.apicast"
+          },
+          {
+            "name": "apicast.policy.http_proxy",
```

Comment:
nit: shouldn't be the apicast policy the last one?

Reply:
They run at different phases so the order is not important. But I updated the PR :smile: